support-frontend/assets/components/footerCompliant/containers/Content.jsx

```diff
@@ -9,40 +9,58 @@ import { brand } from '@guardian/src-foundations/palette';
 import { space } from '@guardian/src-foundations';
 
 type PropTypes = {|
-  paddingTop: boolean,
-  border: boolean,
+  className: string,
+  centred?: boolean,
+  paddingTop?: boolean,
+  border?: boolean,
```

Comment:
Just wondering whether the interface might be cleaner with an appearance object getting passed in here rather than lots of css-related props.